tool/net/lfuncs.c

```diff
@@ -829,6 +829,52 @@ int LuaVisualizeControlCodes(lua_State *L) {
   return LuaCoder(L, VisualizeControlCodes);
 }
 
+int LuaUuidV4(lua_State *L) {
```

Comment:
As discussed, the explicitly unrolled loop here makes the code quite a bit uglier. I would really question whether we're deriving worthwhile benefit from the performance differences at this level.